django/advanced-guides/staging-environments-with-github-actions.html.md

```diff
@@ -0,0 +1,336 @@
+---
+title: Staging environments with GitHub actions
+layout: framework_docs
+order: 1
+objective: How to create staging environments on the Fly.io with GitHub actions.
+author: mariusz
```

Comment:
I believe we can remove `author` from the docs - not sure if they have a way to add multiple authors in case other people update the docs.

It's not consistent within the Frameworks docs but most of the pages don't have author - all but Elixir `Advanced guides` don't have author and those guides were added ~1-2 years ago.

<img width="766" alt="Screenshot 2024-01-15 at 14 29 10" src="https://github.com/superfly/docs/assets/13651115/0aea3386-f8b7-412f-8e57-83019466a702">

Reply:
Removed.